lib/src/main/java/io/ably/lib/transport/Defaults.java

```diff
@@ -1,6 +1,6 @@
 package io.ably.lib.transport;
 
-import io.ably.BuildConfig;
+import io.ably.generated.BuildConfig;
```

Comment:
This change should be reflected in the API updating guide as mentioned in this [comment from another PR](https://github.com/ably/ably-java/pull/768#discussion_r881591978)